sycl/test-e2e/DeviceArchitecture/device_architecture_on_host.cpp

```diff
@@ -0,0 +1,20 @@
+// UNSUPPORTED: cpu, cuda, hip, esimd_emulator
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
+
+int main() {
```

Comment:
Note for reviewers: this test checks that we can compile without any problem with two new APIs, and just checks that `piDeviceGetInfo` is called underneath - so we do not check the correctness of the actual results for these functions as we will have different results on different CI machines

Reply:
Could you please add FileCheck to this test because currently it doesn't check that piDeviceGetInfo is called underneath.